### ISSUE TITLE
train pre-trained model with custom dataset

### ISSUE THREAD

**Issue by tjdgus6255** [open-mmlab/mmsegmentation#2107]
Hello. I got a question.

I trained pre-trained model with custom dataset.

And it works well. But there are a problem.

When I run new model ( pre-trained model + custom dataset), it could not recognize the objects detectable before. 

It is supposed to be like that? or I missed something for training? 

my command line for training here:
python tools/train.py configs/deeplabv3/deeplabv3_r101-d8_512x512_4x4_320k_coco-stuff164k_mydata.py --work-dir custom_model --load-from checkpoint/deeplabv3_r101-d8_512x512_4x4_320k_coco-stuff164k.pth 

If you want details, you can see here #2081

**Comment by MengzhangLI** [comment]
You can check detailed log files between (1) `I trained pre-trained model with custom dataset.` and (2) `I run new model ( pre-trained model + custom dataset)`. Perhaps some setttings in these two configs are different.

**Comment by tjdgus6255** [comment]
> You can check detailed log files between (1) `I trained pre-trained model with custom dataset.` and (2) `I run new model ( pre-trained model + custom dataset)`. Perhaps some setttings in these two configs are different.

you mean I should check pre-trained model config and new model config?

**Comment by MengzhangLI** [comment]
You could try to keep `(2) I run new model ( pre-trained model + custom dataset)` the same as `(1) I trained pre-trained model with custom dataset.` and try again.

**Comment by tjdgus6255** [comment]
> You could try to keep `(2) I run new model ( pre-trained model + custom dataset)` the same as `(1) I trained pre-trained model with custom dataset.` and try again.

I'm sorry for bothering you, but I couldn't understand exactly what you said.

I think you mean, I should train with original config file(no change num_classes, reduce_zero_label, samples_per_gpu)

Is that right?

**Comment by MengzhangLI** [comment]
No, I mean you should keep your parameters in config file (especially those related with the dataset) the same with checkpoints you already trained when you run new model.

**Comment by tjdgus6255** [comment]
I used same config file.(train and run) 

<my dataset  /  coco-stuff164k>
For dataset config, 
I just changed 'samples_per_gpu = 4' into 'samples_per_gpu = 2' 
and I changed 'dict(type='LoadAnnotations'),' into 'dict(type='LoadAnnotations', reduce_zero_label=True),'

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
